src/plugins/intel_gpu/src/graph/include/primitive_inst.h

```diff
@@ -195,14 +184,17 @@ class primitive_inst {
     const std::unordered_map<size_t, std::tuple<int64_t, size_t>>& get_profiling_data() const { return _profiling_data; }
     const std::unordered_map<size_t, instrumentation::perf_counter_key>& get_profiling_info() const { return _profiling_info; }
 
-    layout get_node_output_layout() const { return _node_output_layout; }
+    layout get_node_input_layout() const { return _impl_params->get_input_layout(0); }
+    layout get_node_output_layout() const { return _impl_params->output_layout; }
```

Comment:
Currently it seems to be used only by onednn case. However, it would be better to be in more general way, 

layout get_node_input_layout(int32_t idx = 0)

Also, since it is using impl_param->layout instead of node_layout, so it is no longer "_node_".

Reply:
I renamed it as you guided. Thank you!